.binder/environment.yml

```diff
@@ -0,0 +1,9 @@
+dependencies:
+  - python=3.11
```

Comment:
I would like to try, if `python=3.12` works on mybinder. But at the moment, mybinder.org is unavailable. Will try again later.

Reply:
Does not work.